tools/bin/release_version.sh

```diff
@@ -44,15 +36,3 @@ SUB_BUILD=PLATFORM ./gradlew publish
 VERSION=$NEW_VERSION GIT_REVISION=$GIT_REVISION docker-compose -f docker-compose.build.yaml build
 VERSION=$NEW_VERSION GIT_REVISION=$GIT_REVISION docker-compose -f docker-compose.build.yaml push
 echo "Completed building and publishing..."
-
-echo
-echo "Changelog:"
```

Comment:
Not blocking but nice to have:

It'd be cool if the changelog could also be put in the PR. I imagine you'd have to compute the changelog, save it to an output variable and then read it in the PR creation step?

Reply:
This is done directly in the `pr_body.sh` now.